### PR TITLE
NAS-125756 / 24.04 / Hide Restart and other options from limited users

### PR DESCRIPTION
**Pull request by undsoft**
Testing: create a limited role and observe that Restart, Power Off and Reset to defaults option from General settings are not there.

### CODE REVIEW

**Review by RehanY147 — CHANGES_REQUESTED** [review]
Soon as I login with root, UI crashes with these errors in the console
image

**Comment by undsoft** [comment]
Are you on fresh middleware?

File: src/app/modules/layout/components/topbar/power-menu/power-menu.component.html [hunk 1]
```diff
@@ -14,12 +14,24 @@
...
+    ixTest="restart"
+    (click)="onReboot()"
+  >
     <ix-icon name="replay"></ix-icon>
     {{ 'Restart' | translate }}
   </button>
 
-  <button name="power-shut-down" mat-menu-item ixTest="shut-down" (click)="onShutdown()">
+  <button
+    *ngIf="isSysAdmin$ | async"
```
Comment (RehanY147) [inline src/app/modules/layout/components/topbar/power-menu/power-menu.component.html:29]: We could put both buttons with the same `ngIf` into one `ng-container`

**Review by RehanY147 — APPROVED** [review]
image
The styles are a little messed up but looks good otherwise

**Pull request merged**